test/__mocks__/pack/mockPackSounds.json

```diff
@@ -0,0 +1 @@
+{"count":2,"next":null,"previous":null,"results":[{"id":156417,"name":"2012_05_2012_Siamang_gibbon.wav","tags":["gibbon","Siamang","zoo","Hylobates-syndactylus","rutting-call","monkey"],"license":"http://creativecommons.org/licenses/by/3.0/","username":"reinsamba"},{"id":156416,"name":"2012_05_23_European_wildcat.wav","tags":["wildcat","miaou","miaow","mew","cat","rutting-call","meow"],"license":"http://creativecommons.org/licenses/by/3.0/","username":"reinsamba"}]}
```

Comment:
`test/__mocks__/x/mockX` seems redundant, duplicate context. This should either be `test/__mocks__/pack/sounds.json` or `test/__mocks__/pack-sounds.json`. I think the latter is better for now.

Reply:
Okay. I believe cb09843 addresses this.